statsmodels/base/tests/test_generic_methods.py

```diff
@@ -209,6 +379,11 @@ def setup(self):
         y = x + np.random.randn(x.shape[0])
         self.results = sm.OLS(y, x).fit()
 
+    def test_zero_constrained(self):
+        # override, we cannot remove the only regressor
+        #raise SkipTest
```

Comment:
pls delete commented-out.

If this test is being skipped, it should be explicit.